src/ekore/operator_matrix_elements/polarized/space_like/as1.py

```diff
@@ -1,43 +1,39 @@
 r"""
-This module contains the |NLO| |OME| (OMEs) in the polarized case
-for the matching conditions in the |VFNS|.
-Heavy quark contribution for intrinsic evolution are not considered for the polarized case at this order.
-The matching conditions for the |VFNS| at :math:`\mu_F^2 \neq m_H^2`
-are provided in :cite:`Bierenbaum_2023`.
+This module contains the |NLO| |OME| (OMEs) in the polarized case for the matching conditions in the |VFNS|.
+Heavy quark contribution for intrinsic evolution are not considered for the polarized case.
```

Comment:
```suggestion

Heavy quark contribution for intrinsic evolution are not considered for the polarized case.
```
I think, the new line is needed to separate the description (check the sphinx output (both terminal and browser))